src/Picqer/Financials/Moneybird/Entities/SalesInvoice.php

```diff
@@ -200,4 +200,27 @@ public function duplicateToCreditInvoice()
 
         return $this->makeFromResponse($response);
     }
+
+    /**
+     * Add Attachment to this invoice
+     *
+     * @param string $file
+     *
+     * @return \Picqer\Financials\Moneybird\Entities\SalesInvoice
+     *
+     * @throws \Picqer\Financials\Moneybird\Exceptions\ApiException
+     */
+    public function addAttachment($file)
+    {
+        $this->connection()->upload($this->endpoint . '/' . $this->id . '/attachments', [
+            'multipart' => [
+                [
+                    'name' => 'file',
+                    'contents' => fopen($file, 'r'),
```

Comment:
I think we should just pass the file contents to the `addAttachment` method. This way anyone who implements it has their own choice of how to get the contents. For example if someone has contents in a database, they would query it and pass it to the method.

Reply:
Thanks!